src/actions/ast/setPausePoints.js

```diff
@@ -29,10 +30,10 @@ function compressPausePoints(pausePoints) {
 async function mapLocations(pausePoints, source, sourceMaps) {
   const sourceId = source.id;
   return mapPausePoints(pausePoints, async ({ types, location }) => {
-    const generatedLocation = await sourceMaps.getGeneratedLocation(
-      { ...location, sourceId },
-      source
-    );
+    const generatedLocation = await getGeneratedLocation(sourceMaps, source, {
```

Comment:
we want to use this version of the helper because we will avoid making calls to the worker when it is already generated...